invisible_cities/io/dst_io_test.py

```diff
@@ -62,6 +71,17 @@ def test_load_dsts_single_file(KrMC_kdst):
                             False  , rtol=1e-5)
 
 
+def test_load_dsts_single_file_event_list(KrMC_kdst):
```

Comment:
if is not too much of a hassle (and test files are already in the test dataset) maybe do `load_dsts`for 2 files?